linera-views/src/scylla_db.rs

```diff
@@ -196,9 +196,13 @@ impl ScyllaDbClientInternal {
         client: &ScyllaDbClientPair,
         batch: Batch,
     ) -> Result<(), ScyllaDbContextError> {
-        // We cannot do direct writing of the batch as deletion of key followed by insertion
-        // yield incoherent results.
-        // Thus the batch is first simplified and then the DeletePrefix that collides are removed.
+        // We cannot directly write the batch because if a delete is followed by a write then
+        // the delete takes priority. See the sentence "The first tie-breaking rule when two
+        // cells have the same write timestamp is that dead cells win over live cells"
+        // from https://github.com/scylladb/scylladb/blob/master/docs/dev/timestamp-conflict-resolution.md
+        //
+        // Thus the batch is first simplified. Then when a put operation is colliding after
+        // a delete prefix then we are accessing the prefix and removing the keys.
```

Comment:
I think I understand what you're saying but the sentence is still not super clear. Maybe just remove this paragraph since the previous one explains the issue well.

Reply:
Ok.